example/src/App.js

```diff
@@ -76,9 +77,18 @@ import {
   ShipperRangeSlider
 } from 'redspher-components'
 import 'redspher-components/dist/index.css'
-import { Button, MenuItem, Radio, Select, Chip } from '@material-ui/core'
+import { Button, MenuItem, Radio, Select, Chip, createStyles, makeStyles } from '@material-ui/core'
 import { Pagination } from '@material-ui/lab'
 
+const useStyles = makeStyles((theme) =>
+  createStyles({
+    root: {
+      width: 300 + theme.spacing(3) * 2,
+      paddingLeft: theme.spacing(2)
```

Comment:
```suggestion
      paddingLeft: theme.spacing(2)
```
      marginLeft: theme.spacing(2)

Reply:
```suggestion
      marginLeft: theme.spacing(2)
```